plonk-core/src/proof_system/proof.rs

```diff
@@ -33,13 +33,14 @@ use ark_serialize::{
 };
 use merlin::Transcript;
 
+use super::pi::PI;
+
 /// A Proof is a composition of `Commitment`s to the Witness, Permutation,
-/// Quotient, Shifted and Opening polynomials as well as the
-/// `ProofEvaluations`.
+/// Quotient, Shifted and Opening polynomials as well as the `ProofEvaluations`.
 ///
-/// It's main goal is to allow the `Verifier` to
-/// formally verify that the secret witnesses used to generate the [`Proof`]
-/// satisfy a circuit that both [`Prover`](super::Prover) and
+/// It's main goal is to allow the [`Verifier`](super::Verifier) to formally
+/// verify that the secret witnesses used to generate the [`Proof`] satisfy a
```

Comment:
not sure if "formally verify" is the right language here

Reply:
I agree, in fact I would remove all the paragraph. WDYT?

Reply:
yeah let's get rid of it